docs/api/actions.md

```diff
@@ -159,6 +159,26 @@ is rejected or cancelled with the same payload.
 When all steps of the generator complete, the payload of the parent
 action will be the resolved payload of the final action.
 
+#### Yielding Actions in Parallel
+
+By yielding an array of actions, you can wait for multiple actions to
+complete before continuing:
+
+```javascript
+function getUser (id) {
+  return fetch(`/users/${id}`)
+}
+
+function getUsers (ids) {
+  return function * (repo) {
+    yield ids.map(id => repo.push(getUser, id))
+  }
+}
+```
+
+If all actions resolve or cancel, the generator sequence
+continues.
```

Comment:
huh, not actually sure where this update came from. perhaps something you had in the `repo-all` branch that is no longer there. either way, will update to axios and keep.